pages/contract-address-calculator/index.page.tsx

```diff
@@ -0,0 +1,102 @@
+import { getContractAddress } from '@ethersproject/address';
```

Comment:
Could you also write the tests for this tool? That's a must-have for tools.

Reply:
@olehmisar Everything is okay, but the `contract-address-calculator` page is still missing the tests. Check other pages, all of them have tests and that's a must-have. Thanks!